.changelog/2487.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+control-plane: fixed issue where a HTTPRoute would break if a `URLRewrite` filter was defined.
```

Comment:
This is a new feature for 1.2 correct? If so I don't think we need to include a changelog since the bug was never live

Reply:
Deleting!